.github/workflows/shared-run-e2e-tests.yaml

```diff
@@ -155,11 +173,65 @@ jobs:
           OPERATOR_VERSION: ${{ inputs.operator_image_version }}
           E2E_VERSION: ${{ inputs.e2e_image_version }}
           OPERATOR_TYPE: ${{ inputs.operator_type }}
+          BORE_URL_CORE: ${{ steps.bore.outputs.BORE_URL_CORE }}
+          BORE_URL_OPTOUT: ${{ steps.bore.outputs.BORE_URL_OPTOUT }}
+          BORE_URL_LOCALSTACK: ${{ steps.bore.outputs.BORE_URL_LOCALSTACK }}
+        run: |
+          cd e2e && bash ../uid2-shared-actions/scripts/prepare_resources_for_e2e_docker_compose.sh
+
+      - name: Start GCP private operator
+        id: start_gcp_private_operator
+        if: ${{ inputs.operator_type == 'gcp'}}
+        uses: IABTechLab/uid2-shared-actions/actions/start_gcp_private_operator@kcc-UID2-2339-create-test-pipeline-for-gcp
```

Comment:
I will update the branch name to v2 once https://github.com/IABTechLab/uid2-e2e/pull/6 is merged